src/selectors/per-thread/markers.js

```diff
@@ -57,18 +57,15 @@ export function getMarkerSelectorsPerThread(
    * variants of the selectors that are created for specific views that have been
    * omitted, but the ordered steps below give the general picture.
    *
-   * 1. _getRawMarkerTable - Get the RawMarkerTable from the current thread.
-   * 2. getProcessedRawMarkerTable - Process marker payloads out of raw strings, and
-   *                                 other future processing needs. This returns a
-   *                                 RawMarkerTable still.
-   * 3a. _getDerivedMarkers        - Match up start/end markers, and start
+   * 1. _getRawMarkerTable         - Get the RawMarkerTable from the current thread.
+   * 2a. _getDerivedMarkers        - Match up start/end markers, and start
    *                                 returning the Marker[] type.
-   * 3b. _getDerivedJankMarkers    - Jank markers come from our samples data, and
+   * 2b. _getDerivedJankMarkers    - Jank markers come from our samples data, and
    *                                 this selector returns Marker structures out of
    *                                 the samples structure.
-   * 4. getFullMarkerList          - Concatenates and sorts all markers coming from
+   * 3. getFullMarkerList          - Concatenates and sorts all markers coming from
    *                                 different origin structures.
-   * 5. getFullMarkerListIndexes   - From the full marker list, generates an array
+   * 4. getFullMarkerListIndexes   - From the full marker list, generates an array
    *                                 containing the sequence of indexes for all markers.
    * 5. getCommittedRangeFilteredMarkerIndexes - Apply the committed range.
```

Comment:
ah ah, looks like we couldn't count!

Reply:
It's one of the hard problems in computer science.